@stellar/freighter-api/src/isConnected.ts

```diff
@@ -1,3 +1,9 @@
 import { requestConnectionStatus } from "@shared/api/external";
 
-export const isConnected = () => requestConnectionStatus();
+export const isConnected = () => {
+  if (window?.freighter) {
+    return window.freighter;
+  }
```

Comment:
miraculously, all the other api methods still work fine with the 3.0.0 switch, so just adding a little check here to make this method work with both versions